### PR TITLE
THREAT-3 pySigma Pipeline: SentinelOne

### PR DESCRIPTION
**Pull request by akozlovets098**
pySigma Pipeline: SentinelOne

### CODE REVIEW

File: sigma/pipelines/panther/sentinelone_panther_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+
+from sigma.pipelines.panther.sdyaml_transformation import SdYamlTransformation
+
+
+def sentinelone_panther_pipeline() -> ProcessingPipeline:
+    translation_dict = {
+        "process_creation": {
+            "ProcessId": "tgt_process_pid",
+            "Image": "tgt_process_image_path",
+            "Description": "tgt_process_displayName",  # Not sure whether this should be Description or Product???
```
Comment (le4ker) [inline sigma/pipelines/panther/sentinelone_panther_pipeline.py:18]: Does this have to be one of those two?

Comment (akozlovets098) [reply sigma/pipelines/panther/sentinelone_panther_pipeline.py:18]: To be honest, I'm not sure, but these are logical options for the field `Description`

Comment (le4ker) [reply sigma/pipelines/panther/sentinelone_panther_pipeline.py:18]: Why not `ProcessName` then?

Comment (le4ker) [reply sigma/pipelines/panther/sentinelone_panther_pipeline.py:18]: This comment is from [here](https://github.com/7RedViolin/pySigma-backend-sentinelone/blob/master/sigma/pipelines/sentinelone/sentinelone.py#L41), I thought it was yours 😅 

Let's leave the same mapping on our end.

**Pull request merged**